### ISSUE TITLE
Install `libnss` when `puppeteer` is used

### ISSUE THREAD

**Issue by Milo123459** [railwayapp/nixpacks#294]
We need to install `libnss` via `apt`. It is probably worth overhauling the current solution to check if packages are installed and make one-function, purpose-built for this.

**Comment by thawankeane** [comment]
It can be done with `NIXPACKS_APT_PKGS` environment variable.

See https://nixpacks.com/docs/config#environment-variables

**Comment by JakeCooper** [comment]
Would be good to, if we can detect it, install the required application packages

@wyzlle got it working using. This is a great first issue for new contributors!

```
NIXPACKS_APT_PKGS="libnss3 libatk1.0-0 libatk-bridge2.0-0 libcups2 libgbm1 libasound2 libpangocairo-1.0-0 libxss1 libgtk-3-0" 
```

**Comment by JakeCooper** [comment]
This has been resolved by #416
